lib/numbers.md

```diff
@@ -5,20 +5,26 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+The two basic kinds of numbers in Ruby are integers (whole numbers) and floats (numbers with decimal points).
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+Some common operators would be addition '+', subtraction '-', multiplication '*', division '/', exponentiation '**', and modulo '%'.
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+A '+' will combine the contents of strings added together.  For example, "2" + "7" => "27"
+A '+' with numbers will perform the addition operation.  For example, 2 + 7 => 9
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+Some mathematical operations can be performed on strings, some can not.  
+Strings can be added to other strings, i.e. "20" + "3" => "203".  Here we are simply adding the contents of the strings.
+Strings can be multiplied by a number, i.e. "20" * 3 => "202020". Here we are telling Ruby to add the contents of the string "20" + "20" + "20".
+Note: for this question the string must come first, i.e. 3 * "20" results in an error because we can not multiply a number by a string.
+All other mathematical operations with respect to strings will result in an error because they do not follow any logic.
```

Comment:
What if we wanted to divide it by 3? Or divide `"5"` by 2? Assuming we wanted the exact [quotient](https://www.mathsisfun.com/definitions/quotient.html), would we still use `.to_i`?